test/Spec.hs

```diff
@@ -71,8 +72,21 @@ main =
     -- , bitmaskSpec @Word64
     -- , bitmaskSpec @Word
     , runSpec
+    , unitTests
     ]
 
+unitTests :: TestTree
+unitTests = testGroup "Unit tests"
+  [ -- Check that https://github.com/haskell/random/issues/53 does not regress
+
+    testCase "Subnormal generation respects upper bound" $
+    [] @?= (filter (>4.0e-45) $ take 10 $ randomRs (0,(4.0e-45::Float)) $ mkStdGen 0)
```

Comment:
```suggestion
    [] @?= (filter (>4.0e-45) $ take 100000 $ randomRs (0,(4.0e-45::Float)) $ mkStdGen 0)
```

... still runs imperceptably fast :)

10 just seems a bit low for this sort of test, not that I can say objectively why 100000 is necessarily better.